wiki/Behavior.md

```diff
@@ -26,3 +26,47 @@ The name is generated as follows:
 See the [Migration System] page.
 
 [Migration System]: https://github.com/recyclarr/recyclarr/wiki/Migration-System
+
+## Radarr
+
+### Custom format synchronization is broken up into three categories
```

Comment:
How about this instead?

```md
## Radarr

### Custom Format Synchronization

Custom format synchronization is broken up into three categories:
```

Reply:
I agree, I'll make the change and repush.

Reply:
Sorry to keep nitpicking; don't forget capitalization:

    ### Custom Format Synchronization

Reply:
Done, I'll make sure to pay closer attention to headers and camel case the headers every time.